src/Sulu/Component/Content/SmartContent/QueryBuilder.php

```diff
@@ -329,7 +329,7 @@ protected function buildTypesWhere($types, string $languageCode)
     {
         $sql2Where = [];
         foreach ($types as $type) {
-            $sql2Where[] = 'page.[i18n:' . $languageCode . '-template] = ' . $type;
+            $sql2Where[] = \sprintf("page.[i18n:%s-template] = '%s'", $languageCode, $type);
```

Comment:
we always use single quotes like in the example above in the code we should just escape the quotes:

```suggestion
            $sql2Where[] = \sprintf('page.[i18n:%s-template] = \'%s\'', $languageCode, $type);
```